frontend/sw360-portlet/src/main/webapp/html/include/searchfield.jspf

```diff
@@ -0,0 +1,89 @@
+<%--
+ ~ Copyright Siemens AG, 2017.
+ ~ Part of the SW360 Portal Project.
+ ~
+ ~ SPDX-License-Identifier: EPL-1.0
+ ~
+ ~ All rights reserved. This program and the accompanying materials
+ ~ are made available under the terms of the Eclipse Public License v1.0
+ ~ which accompanies this distribution, and is available at
+ ~ http://www.eclipse.org/legal/epl-v10.html
+ --%>
+ <div id="component-searchField" class="content1">
+    <table>
+        <thead>
+        <tr>
+            <th class="infoheading">
+                Display Filter
+            </th>
+        </tr>
+        </thead>
+        <tbody style="background-color: #f8f7f7; border: none;">
+        <tr>
+            <td>
+                <input type="text" class="searchbar" value="" />
+                <br/>
+                <input type="button" class="searchbutton" name="searchBtn" value="Search" />
+            </td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+<script>
+define('component/searchfield', ['jquery'], function(jquery) {
```

Comment:
Modules are defined using the define function. Dependencies can also be given that are loaded to execute the module.